xc/xenops_server_xen.ml

```diff
@@ -1094,6 +1094,13 @@ module VM = struct
 		)
 	) Newest task vm
 
+	let set_vcpus task vm target = on_domain (fun xc xs _ _ di ->
```

Comment:
The parameter `target` is called `ncpus` in the C code and interface. I would consider using that name here, too, because it seems to me more specific that `target`.

Reply:
Thanks, rename `target` to `ncpus`.